cpp/dolfinx/fem/FiniteElement.h

```diff
@@ -189,6 +189,11 @@ class FiniteElement
   /// @return True if interpolation is an identity operation
   bool interpolation_ident() const noexcept;
 
+  /// Check if the map from the reference to a physical cell for this element is
```

Comment:
I don't quite follow. When is the map ever the identity? The reference and physically cells have different geometry.

Reply:
I mean the push forward or pull back map is the identity. Perhaps a better name is needed

Reply:
@jpdean might have a suggestion for a name.

Reply:
I've updated the docstring to say this is the push forward/pull back map we're talking about. Happy to take suggestions for a better name

Reply:
Sorry, I didn't see this! I think calling it a pullback/pushforward is fine.